### PR TITLE
Feature/ju vp 001 maria filterdata comboboxfilter

### PR DESCRIPTION
**Pull request by MariaAT18**

### CODE REVIEW

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java [hunk 1]
```diff
@@ -21,19 +25,29 @@
...
+                ListValidVideos.getInstance().setFormatSelected(itemSelected);
+            }
+        });
     }
 
     /**
      * Add a list of items to the comboBox
-     * 
+     *
      * @param itemsList
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/view/commons/ComboBox.java]: You could put the description of the ItemsList parameter

File: app/src/main/java/ju/video/player/commons/FilterFiles.java [hunk 2]
```diff
@@ -56,15 +59,33 @@ public List<String> getListFiles() throws IOException {
...
         }
         return listFilesName;
     }
 
+    /**
+     * Verify the file belongs to the format selected in the Combobox.
+     *
+     * @param file
+     * @param formatSelected
+     * @return
```
Comment (junior0123) [inline app/src/main/java/ju/video/player/commons/FilterFiles.java]: You could put the description of parameters and description of return

**Review by junior0123 — CHANGES_REQUESTED** [review]

File: app/src/main/java/ju/video/player/commons/FilterFiles.java (hunk 2 above)
Comment (MariaAT18) [reply app/src/main/java/ju/video/player/commons/FilterFiles.java]: Done

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java [hunk 3]
```diff
@@ -1,15 +1,19 @@
 /**
  * Copyright (c) 2022 Jala University.
- *
+ * <p>
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/commons/ComboBox.java:3]: Only a question, why yo add this <p>, what is its meaning?

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java [hunk 4]
```diff
@@ -21,19 +25,29 @@
...
 
     /**
      * Add a list of items to the comboBox
-     * 
+     *
      * @param itemsList
      */
     private void addItems(String[] itemsList) {
-        for(int index = 0; index < itemsList.length; index++) {
+        for (int index = 0; index < itemsList.length; index++) {
```
Comment (AdriOli134) [inline app/src/main/java/ju/video/player/view/commons/ComboBox.java:50]: thanks for fix it

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java (hunk 1 above)
Comment (MariaAT18) [reply app/src/main/java/ju/video/player/view/commons/ComboBox.java]: done

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java (hunk 3 above)
Comment (MariaAT18) [reply app/src/main/java/ju/video/player/view/commons/ComboBox.java:3]: IntelliJ IDE add this tag. I think that this is to have a Line break in the document.

File: app/src/main/java/ju/video/player/view/commons/ComboBox.java (hunk 4 above)
Comment (MariaAT18) [reply app/src/main/java/ju/video/player/view/commons/ComboBox.java:50]: welcome

**Review by junior0123 — APPROVED** [review]

**Review by AdriOli134 — CHANGES_REQUESTED** [review]
Please, I forgot in the previous reviews, but don't forget to add the documentation of each class, up to the class name.

**Review by AdriOli134 — APPROVED** [review]

**Pull request merged**